docs/source/format/driver_manifests.rst

```diff
@@ -351,7 +351,8 @@ to control which directories will be searched for manifests, with the behavior b
 
        The type :c:type:`AdbcLoadFlags` is a set of bitflags to control the directories to be searched. The flags are
 
-       * :c:macro:`ADBC_LOAD_FLAG_SEARCH_ENV` - search the environment variable ``ADBC_CONFIG_PATH``
+       * :c:macro:`ADBC_LOAD_FLAG_SEARCH_ENV` - search the directory paths in the environment variable
+       ``ADBC_CONFIG_PATH`` and (when built or installed with conda) search in the conda environment
```

Comment:
```suggestion
       * :c:macro:`ADBC_LOAD_FLAG_SEARCH_ENV` - search the directory paths in the environment variable
         ``ADBC_CONFIG_PATH`` and (when built or installed with conda) search in the conda environment
```

I think these need to be indended for reST to recognize it as part of the same list

Reply:
Fixed in e32e72a